v2.0/demo-json-support.md

```diff
@@ -0,0 +1,186 @@
+|              | )                                               |
++--------------+-------------------------------------------------+
+~~~
+
+## Step 6. Run the code
+
+Now that you have a database, user, and a table, let's run code to insert rows into the table.
+
+The code queries the [Reddit API](https://www.reddit.com/dev/api/) for posts in [/r/programming](https://www.reddit.com/r/programming/). The Reddit API only returns 25 results per page; however, each page returns an `"after"` string that tells you how to get the next page. Therefore, the program does the following in a loop:
+
+1. Makes a request to the API
+2. Inserts the results into the table and grabs the `"after"` string
+3. Uses the new `"after"` string as the basis for the next request
+
+Download the <a href="https://raw.githubusercontent.com/cockroachdb/docs/master/_includes/json/json-sample.go" download><code>json-sample.go</code></a> file, or create the file yourself and copy the code into it:
+
+{% include copy-clipboard.html %}
+~~~ go
+{% include json/json-sample.go %}
+~~~
+
+From a new terminal window, navigate to your sample code file and run it:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ go run json_sample.go
+~~~
+
+The program will take a few minutes to insert rows into your table. If you want to verify that rows of data were inserted into your table,  use the [built-in SQL client](use-the-built-in-sql-client.html):
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT count(*) FROM programming;
+~~~
+~~~
++-------+
+| count |
++-------+
+|  7500 |
++-------+
+~~~
+
+## Step 7. Query the data
+
+Once there is data in your table, you can query the data. For this example, retrieve all the entries where the link is pointing to somewhere on YouTube. Specifically, all the entries where `object.data.domain` is `"youtube.com"`. To do that, run:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT id FROM programming WHERE posts @> '{"data": {"domain": "youtube.com"}}';
+~~~
+~~~
+(334 rows)
+
+Time: 105.877736ms
+~~~
+
+{{site.data.alerts.callout_info}}Since you are querying live data, your results for this and the following steps may vary from the results documented in this tutorial.{{site.data.alerts.end}}
+
+## Step 8. Create an inverted index to optimize performance
+
+The query in the previous step took 105.877736ms. An [inverted index](inverted-indexes.html) will optimize the performance. Let's create an inverted index on the `JSONB` column:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE INVERTED INDEX ON programming(posts);
+~~~
+
+## Step 9. Run the query again
+
+Now that there is an inverted index, the query should run faster. Run the same query:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT id FROM programming WHERE posts @> '{"data": {"domain": "youtube.com"}}';
+~~~
+~~~
+(334 rows)
+
+Time: 28.646769ms
+~~~
+
+After creating the inverted index, the query now takes 28.646769ms.
```

Comment:
nit: "After creating the inverted index" feels a bit redundant. Maybe just expand the second sentence to:

```
Instead of 105.877736ms, the query now takes 28.646769ms.
```